spec/factories/schools.rb

```diff
@@ -0,0 +1,10 @@
+FactoryBot.define do
+  factory :school do
+    name { 'Example School' }
```

Comment:
Please check if you need those brackets in attributes with simple value like: string, bool, number